platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Core.swift

```diff
@@ -23,14 +23,20 @@ public final class Core {
        
         return handleCoreResponse(core_bootstrap(self.languageCode), { _ in })
     }
-    
-    /// Parse the FeatureManifest from the given json
-    /// - Parameter json: Raw JSON to parse the FeatureManifest from
-    /// - Returns: A FeatureManifest if parsing succeded, nil otherwise
-    public func parseFeatureManifest(json: String) -> Result<FeatureManifest, Error> {
-        handleCoreResponse(parse_feature_manifest_from_json(json), mapFeatureManifest)
+
+    /// Loads the feature categories from the give features directory
+    public func loadFeatureCategories(
+        featuresDirectory: String
+    ) -> Result<[FeatureCategory], Error> {
```

Comment:
Why not leave the doc? They were not great, but at least they contained some explanations...